docs/q-and-a.md

```diff
@@ -6,8 +6,12 @@ The Q&A frontend is a finder-frontend feature that walks users through the facet
 
 Any, all or no filters can be selected for each facet, and any question may be skipped. At the end of the process, the user is directed to the given finder with their selections intact.
 
-At the time of writing, this feature has been defined for one finder and relies on a [YAML file](../lib/prepare_business_uk_leaving_eu.yaml). This file defines things like the base path of the Q&A and the underlying finder, as well as questions, question types and descriptions for each question page.
+It emulates the look and feel of a [Smart Answer](https://github.com/alphagov/smart-answers), but the code lives here so that we can generate a business finder URL from the user's answers and have the finder's facets pre-populated.
```

Comment:
I don't know where the previous comment has gone, but I don't know if this the motivation for adding the Q&A here is correct.

I hope it's following the question pages pattern in the design system, (@DilwoarH will know more), but as we can't be sure, I think this paragraph should be removed.

Reply:
The other comment is here: https://github.com/alphagov/finder-frontend/pull/1203#discussion_r296821418

Think this is correct based on what Dilwoar said. ^

Reply:
I think the deleted lines are more relevant. Maybe it should be something like this:

```
The QA feature follows the [question pages pattern from the design system](https://design-system.service.gov.uk/patterns/question-pages/). It uses the answers to each question to generate a URL which preselects facets on a finder.

At the time of writing, this feature has been defined for one finder and relies on a [YAML file](../lib/prepare_business_uk_leaving_eu.yaml). This file defines things like the base path of the Q&A and the underlying finder, as well as questions, question types and descriptions for each question page.
```

^^ @ChrisBAshton @leenagupte Please feel free to reword if you need

Previous comment:
https://github.com/alphagov/finder-frontend/pull/1203#pullrequestreview-253539318

Reply:
Thanks, will amend.